lib/coap/connection.ex

```diff
@@ -163,7 +178,8 @@ defmodule CoAP.Connection do
 
   # TODO: resend reset?
   # TODO: what is the message if the client has to re-request after a processing timeout from the app?
-  defp receive_message(_message, %{phase: :peer_ack_sent} = state), do: state
+  # defp receive_message(_message, %{phase: :peer_ack_sent} = state), do: state
+  # TODO: what about receiving the request for the next chunk?
```

Comment:
Isn't this covered on line 194?

Reply:
Yeah, I'm gonna take a pass at removing commented code and TODOs that are "completed".

Reply:
I probably need this for when it is not multipart, but below 194.